### PR TITLE
itr-5: feat: add modal window

### PR DESCRIPTION
**Pull request by MilaKolasava**
Снимок экрана 2022-07-11 162827

### CODE REVIEW

File: src/components/ModalWindow/ModalWindow.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.6);
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.modalContent {
```
Comment (glebpigulevsky) [inline src/components/ModalWindow/ModalWindow.css:13]: empty line between style selectors

File: src/components/ModalWindow/ModalWindow.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./ModalWindow.css";
+
+function ModalWindow() {
+  return (
+    <div className="modalWindow">
+      <div className="modalContent"></div>
```
Comment (glebpigulevsky) [inline src/components/ModalWindow/ModalWindow.tsx]: you can use selfclosing of empty div

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
two minor comments

File: src/components/ModalWindow/ModalWindow.css (hunk 1 above)
Comment (MilaKolasava) [reply src/components/ModalWindow/ModalWindow.css:13]: done

**Review by glebpigulevsky — APPROVED** [review]
good job!

File: src/components/ModalWindow/ModalWindow.tsx (hunk 2 above)
Comment (MilaKolasava) [reply src/components/ModalWindow/ModalWindow.tsx]: done

**Pull request merged**